alacritty/src/config/monitor.rs

```diff
@@ -26,7 +26,7 @@ impl Monitor {
                 // The Duration argument is a debouncing period.
                 let mut watcher =
                     watcher(tx, Duration::from_millis(10)).expect("Unable to spawn file watcher");
-                let config_path = ::std::fs::canonicalize(path).expect("canonicalize config path");
+                let config_path = std::fs::canonicalize(path).expect("canonicalize config path");
```

Comment:
If these are touched, we might as well properly import `std::fs` at the top of the file.